code/modules/power/engine.dm

```diff
@@ -13,11 +13,4 @@
 			if(prob(50))
 				ChangeTurf(src.baseturf)
 		else
-			return
-
-/turf/simulated/floor/engine/blob_act()
-	if (prob(25))
-		ChangeTurf(src.baseturf)
-		qdel(src)
-		return
-	return
```

Comment:
Notable stupid shit found in blob_act()
Floors shouldn't have a blob act, because it'd only be called when attacking something dense above them.